tools/tests/test_cases/test_gemm.cpp

```diff
@@ -171,3 +172,75 @@ TEST_F(DnnlPluginNext_GEMM, LLaMa_v2_case_2)
     params_.allow_fp16_computations = true;
     run();
 }
+
+TEST_F(DnnlPluginNext_GEMM, TransposedCases_Normal_FP32)
```

Comment:
Could you add some more tests with a bit bigger dimensions? Optionally you may want to modify these tests to be parametrized (please have a look at example in "test_gemm_paraetrized.cpp")

Reply:
Added some bigger dimensions to the tests_gemm.cpp
Also modified the parametrized_tests.cpp by adding transpose cases.

Reply:
Please try this one
[abdc](https://oneapi-src.github.io/oneDNN/enum_dnnl_memory_format_tag.html#doxid-structdnnl-1-1memory-1a8e71077ed6a5f7fb7b3e6e1a5a2ecf3fa5e822a0b52c71ce2ff2ecd3aa8ca46fc)            = dnnl_abdc,
If it doesnt work then instead format tag please try to use strides

Reply:
Tried to write a function to swap dimensions and using _format_tag::abdc_ - in dnnl_utils.h